### ISSUE TITLE
AttributeError: 'DETR' object has no attribute 'train'

### ISSUE THREAD

**Issue by dokooh** [autodistill/autodistill#67]
### Search before asking

- [X] I have searched the Autodistill [issues](https://github.com/autodistill/autodistill/issues) and found no similar bug report.

### Bug

I tried to distill Grounding SAM with a DETR object detector and got a weird output thought to report.

Basically, I edited the sample code you have in your doc but I added caption ontology and used the YAML folder to run the train method, first part went well and it downloaded the pretrained ResNet but ran into error on the second line, it seems that somehow it cannot find the train function (see below), I use autodistill_detr-0.1.0:
```

from autodistill_detr import DETR

# load the model
target_model = DETR(ontology)

# train for 10 epochs
target_model.train(DATA_YAML_PATH, epochs=10)

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[36], line 8
      4 target_model = DETR(ontology)
      6 # train for 10 epochs
      7 #target_model.train("./roads", epochs=10)
----> 8 target_model.train(DATA_YAML_PATH, epochs=10)
     10 # run inference on an image
     11 #target_model.predict("./roads/valid/-3-_jpg.rf.bee113a09b22282980c289842aedfc4a.jpg")

AttributeError: 'DETR' object has no attribute 'train'
```

### Environment

- Kaggle jupyter notebook (also saw same behavior on AWS sagemaker)
- GPU: T4 x2
- autodistill 0.1.0
- autodistill_detr-0.1.0

### Minimal Reproducible Example

```
from autodistill.detection import CaptionOntology
from autodistill_detr import DETR

ontology=CaptionOntology({
    "circle": "19",
    "road sign" : "16", 
})

# load the model
target_model = DETR(ontology)

# train for 10 epochs
#target_model.train(DATA_YAML_PATH, epochs=10)
```

### Additional

I opened this issue under autodistill-detr and got no attention, if requested I will reopen the original issue:
https://github.com/autodistill/autodistill-detr/issues/2

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by OlivierNikkori** [comment]
Code for train is already in main (autodistill-detr), is it possible to create a new release ?

**Comment by capjamesg** [comment]
I'm not sure why we didn't publish the `train` code on PyPi! I will publish this tomorrow and post a message here when I have published the release.

**Comment by capjamesg** [comment]
I have published `autodistill-detr==0.1.1`, which includes the `train()` code: https://pypi.org/project/autodistill-detr/

You will need to run `pip install --upgrade autodistill-detr` to use the new changes.

I am going to close this issue now. Let me know if you run into any issues and I can re-open the issue.
